source/includes/common-installation.rst

```diff
@@ -211,7 +211,36 @@ MinIO **does not** support arbitrary migration of a drive with existing MinIO da
 
 .. end-local-jbod-single-node-desc
 
+.. start-storage-requirements-desc
+
+The following requirements summarize the :ref:`minio-hardware-checklist-storage` section of MinIO's hardware recommendations:
+
+Use Local Storage
+   Direct-Attached Storage (DAS) has significant performance and consistency advantages over networked storage (NAS, SAN, NFS).
+   MinIO strongly recommends flash storage (NVMe, SSD) for primary or "hot" data.
+
+Use XFS-Formatting for Drives
+   Deployments using non-XFS filesystems (EXT4, BTRFS, ZFS) tend to have lower performance and may also exhibit unexpected or undesired behavior at scale or under load.
+
+Use Consistent Type of Drive
+   Each :term:`pool` must use the same type (NVMe, SSD)  of drive with identical capacity (e.g. ``N`` TB) . 
+   MinIO does not distinguish drive types and does not benefit from mixed storage types. 
+
+Use Consistent Size of Drive
+   MinIO limits the size used per drive to the smallest drive in the deployment. 
+   For example, if the deployment has 15 10TB drives and 1 1TB drive, MinIO limits the per-drive capacity to 1TB.
+
+Configure Sequential Drive Mounting
+   MinIO uses expansion notation ``{x...y}`` to denote a sequential series of drives when creating the new |deployment|, where all nodes in the |deployment| have an identical set of mounted drives. 
+   Configure drive mounting paths as a sequential series to best support this notation.
+
+Persist Drive Mounting and Mapping Across Reboots
+   Use ``/etc/fstab`` or a similar file-based mount configuration to ensure consistent drive-to-mount mapping across node reboots.
```

Comment:
nit : similar is confusing here since it implies there are multiple avenues in linux, maybe replace with or equivalent for non linux OS